### ISSUE TITLE
[BUG] PhyloSeq creation failed when just concat is used!

### ISSUE THREAD

**Issue by bhagesh-h** [quadram-institute-bioscience/dadaist2#20]
**PhyloSeq creation failed. at /root/miniconda/bin/dadaist2-phyloseqMake line 96.**
I ran three samples together for my analysis, but got phyloseq error while using just concat (-j) option!

**To Reproduce**
Following is the command:
`mkdir -p output/ && dadaist2 --max-loss 0.01 -i input/ -o output/ -d ref/bacterial16S.fasta.gz --trim-primer-for 0 --trim-primer-rev 0 -no-trunc -j -t 16 -m metadata.tsv --verbose`

**Error Log**
```
Dadaist2
[2022-10-27 09:38:53] Ready to log in /mnt/output_04/dadaist.log
[2022-10-27 09:38:53] dadaist2 1.0.2
[2022-10-27 09:38:53] Taxonomy database found: ref/zymo.bacterial16S.cleaned.fasta.gz
 * Input directory: input/
 * Output directory: /mnt/output_04/
 * Metadata: metadata.tsv
 * Reference database: ref/zymo.bacterial16S.cleaned.fasta.gz
 * Threads: 16
 * Temporary directory: /tmp/dadaist2_HYjKoX
 * QC strategy: fastp
[2022-10-27 09:38:53] Checking dependencies
 * RScript: R scripting front-end version 4.0.5 (2021-03-31)
 * Taxonomy: dadaist2-assigntax 1.1.3
 * assign-taxonomy: dadaist2-assigntax 1.1.3
 * clustalo: 1.2.4
 * dada2 (lib): <pass>
 * exporter: dadaist2-exporter 1.4.0
 * fastp: fastp 0.20.1
 * fasttree: FastTree version 2.1.10 Double precision (No SSE3):
 * fu-primers: fu-primers 1.6.0
[2022-10-27 09:39:02] Temporary directory: /tmp/dadaist2_HYjKoX
[2022-10-27 09:39:02] Threads: 16
[2022-10-27 09:39:02] Output directory: /mnt/output_04/
[2022-10-27 09:39:02] Input directory "input/": 3 found (paired-end)
[2022-10-27 09:39:02] (1/3) Processing V1V3aZYMO: fastp
[2022-10-27 09:39:24] 152268/152270 (100%) reads kept.
 * Average insert size: 382 bp
 * Q30: 0.732603
 * Qualified region: [0 - 271]/301, [0 - 191]/301
[2022-10-27 09:39:24] (2/3) Processing V3V4aZYMO: fastp
[2022-10-27 09:39:48] 208536/208536 (100%) reads kept.
 * Average insert size: 458 bp
 * Q30: 0.837994
 * Qualified region: [0 - 296]/301, [0 - 220]/301
[2022-10-27 09:39:48] (3/3) Processing V3V5bZYMO: fastp
[2022-10-27 09:40:11] 175472/175472 (100%) reads kept.
 * Average insert size: 420 bp
 * Q30: 0.758107
 * Qualified region: [0 - 287]/301, [0 - 187]/301
[2022-10-27 09:40:11] Skipping truncation
[2022-10-27 09:40:11] Running DADA2...
[2022-10-27 09:42:29] DADA2 Finished.
[2022-10-27 09:42:29] Converting dada2 taxonomy output: /tmp/dadaist2_HYjKoX/taxonomy.tsv
[2022-10-27 09:42:29] 97 representative sequences found.
[2022-10-27 09:42:29] DADA2 filtered 56.0409% from total 268138 to 150267
readline() on closed filehandle $I at /root/miniconda/bin/dadaist2 line 935.
[2022-10-27 09:42:29] Multiple sequence alignment and tree generation
[2022-10-27 09:43:18] Feature tree generated
[2022-10-27 09:43:18] Exporting MicrobiomeAnalyst
[2022-10-27 09:43:23] PhyloSeq file not generated: /mnt/output_04/R/phyloseq.rds
[2022-10-27 09:43:23] Diagnostics:
 DADAIST2 Import to PhyloSeq
R version 4.0.5 (2021-03-31)
 * Input:  /mnt/output_04/
 * Loading feature table
 * Taxonomy loaded
 * Tree loaded
 * Metadata loaded
 * PhyloSeq: Feature table done
Error in dimnames(x) <- dn :
  length of 'dimnames' [1] not equal to array extent
Calls: tax_table -> tax_table -> .local -> rownames<- -> rownames<-
Execution halted

PhyloSeq creation failed. at /root/miniconda/bin/dadaist2-phyloseqMake line 96.
[2022-10-27 09:43:26] Rhea normalization/alpha finished.
[2022-10-27 09:43:26] Dadaist finished, output files saved:
 * dada-taxonomy-table: /mnt/output_04/taxonomy.txt
 * feature-table: /mnt/output_04/feature-table.tsv
 * features-tree: /mnt/output_04/rep-seqs.tree
 * mba-files: /mnt/output_04/MicrobiomeAnalyst
 * multiple-alignment: /mnt/output_04/rep-seqs.msa
 * rep-seqs: /mnt/output_04/rep-seqs.fasta
 * rhea: /mnt/output_04/Rhea

[2022-10-27 09:43:26] Cleaning up

```

 **Environment:**
 - OS: [e.g. Ubuntu 18.04]
- Dadaist2 version: 1.0.2

**Comment by telatin** [comment]
Thanks for reporting, will check what happened

**Comment by telatin** [comment]
FYI I'm about to release 1.3.0 with some bugfixes, then that issue will be also patched in the upcoming 1.4.0
